node/src/types/deploy.rs

```diff
@@ -598,12 +598,12 @@ impl FromBytes for Deploy {
 
 #[cfg(test)]
 mod tests {
+    use super::*;
+
```

Comment:
This change should be reverted.  The import order is std, third party, other casper crates, this crate.